f5/bigip/__init__.py

```diff
@@ -30,7 +30,7 @@
 
 class BigIP(OrganizingCollection):
     """An interface to a single BIG-IP"""
-    def __init__(self, hostname, username, password, **kwargs):
+    def __init__(self, hostname, username, password, port=443, **kwargs):
```

Comment:
We're using the keyword args default params pattern here, soo...

Instead of having `port=443` in the param list for the constructor, move it down into the "kwargs.pop"s.

So the whole param list should be:

`def __init__(self, hostname, username, password, **kwargs):`

and then...   see next line note...